setup.cfg

```diff
@@ -1,2 +1,35 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = Flask-DebugToolbar
+version = 0.12.dev0
+author = Michael van Tellingen
+author_email = michaelvantellingen@gmail.com
+maintainer = Matt Good
+maintainer_email = matt@matt-good.net
+description = A toolbar overlay for debugging Flask applications.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+keywords = flask, debug, toolbar
+url = https://github.com/flask-debugtoolbar/flask-debugtoolbar
+project_urls =
+    Changelog = https://github.com/jeffwidman/cqlsh#changelog
+    Documentation = https://github.com/flask-debugtoolbar/flask-debugtoolbar/blob/master/CHANGES.rst
```

Comment:
These were both incorrect, fixed in https://github.com/flask-debugtoolbar/flask-debugtoolbar/commit/5eea25882c251457088830ea1e1a166c64aaec0f